src/converters/number-compare-converters.js

```diff
@@ -14,23 +38,118 @@ function parseArgs(limit, trueVal, falseVal, valueToCompare, matchString) {
     }
 }
 module.exports = {
+    /**
+     * Convert the model variable to true, or other values passed to the converter, based on whether the model variable is greater than the limit.
```

Comment:
"converts the model variable" sounds a bit like it's changing the value in the model, maybe "converts the model variable output" would be more accurate?